server/app.js

```diff
@@ -55,6 +56,8 @@ app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/appointment", appointmentRouter);
 app.use("/meeting", meetingRouter);
+app.use("/availability", availabilityRouter);
```

Comment:
May I know the purpose of having two different endpoints point to the same _availabilityRouter_. I can't see any other references to it

Reply:
Removed the duplicate, I forgot to remove this when resolving merge conflicts.

Reply:
@bonnieli  Looks good from my side now